static/app/stories/playground/slideOverPanel.tsx

```diff
@@ -14,11 +15,13 @@ export function SlideOverPanelPlayground() {
     <Fragment>
       <Button onClick={() => setIsPanelOpen(true)}>Open Panel</Button>
 
-      <SlideOverPanel open={isPanelOpen} position="right">
-        <Container border="primary" height="100%" padding="md">
-          <Button onClick={() => setIsPanelOpen(false)}>Close Panel</Button>
-        </Container>
-      </SlideOverPanel>
+      {isPanelOpen && (
+        <SlideOverPanel position="right">
```

Comment:
@gggritso how would you feel about about making `SlideOverPanel` extend the Container component? Does that make sense and would it be helpful? The reason I ask is because having a child container with border=primary feels like it's specifying border on the wrong component